src/object-store/src/cache_policy.rs

```diff
@@ -89,12 +92,16 @@ impl<I: Accessor, C: Accessor> LayeredAccessor for LruCacheAccessor<I, C> {
 
         match self.cache.read(&cache_path, OpRead::default()).await {
             Ok((rp, r)) => {
+                increment_counter!(OBJECT_STORE_LRU_CACHE_HIT);
+
                 // update lru when cache hit
                 let mut lru_cache = lru_cache.lock().await;
                 lru_cache.get_or_insert(cache_path.clone(), || ());
                 Ok(to_output_reader((rp, r)))
             }
             Err(err) if err.kind() == ErrorKind::NotFound => {
+                increment_counter!(OBJECT_STORE_LRU_CACHE_MISS);
```

Comment:
```suggestion
                increment_counter!(metrics::OBJECT_STORE_LRU_CACHE_MISS);
```

Reply:
Done. See the [comment ](https://github.com/GreptimeTeam/greptimedb/pull/1405#discussion_r1169460752) below.